### ISSUE TITLE
Cannot name file

### ISSUE THREAD

**Issue by jasongonzales23** [axwaxw/json2csv#2]
While I can get the file to download when using Chrome, the name I assign to the file doesn't take. Chrome labels the file "download.csv" no matter what I do.  I found this SO poster with the same issue: http://goo.gl/FzTR1i

**Comment by secondspace** [comment]
Anybody have a solution for this? I'm having the same problem.

**Comment by axwaxw** [comment]
Yes there does seem to be an issue with the download attribute in HTML5 and certain browsers.  The json2csv package is still working, however.

I will look at other ways to achieve download per the example in the README file that do not rely on HTML5.
